bin/NativeTests/BigIntTest.cpp

```diff
@@ -0,0 +1,191 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "stdafx.h"
+#pragma warning(disable:26434) // Function definition hides non-virtual function in base class
+#pragma warning(disable:26439) // Implicit noexcept
+#pragma warning(disable:26451) // Arithmetic overflow
+#pragma warning(disable:26495) // Uninitialized member variable
+#include "catch.hpp"
+
+#include "BigIntTest.h"
+
+
+namespace BigIntTest
+{
+    TEST_CASE("Init_Compare", "[BigIntTest]")
+    {
+        uint32 digits[1];
+        digits[0] = 0x00001111;
+        int32 length = 1;
+        Js::BigInt bi1, bi2;
+        BOOL f;
+        f = bi1.FInitFromRglu(digits, length);
+        Assert(f);
+        f = bi2.FInitFromRglu(digits, length);
+        Assert(f);
+        // Equal
+        int result;
+        result = bi1.Compare(&bi2);
+        CHECK(result == 0);
+        f = bi2.FInitFromBigint(&bi1);
+        Assert(f);
```

Comment:
A slightly more Catch-y way to do this is to use REQUIRE() (REQUIRE stops the program when the condition fails, CHECK allows the execution to continue)